components/library/libxxhash/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2024 <contributor>
+#
+
+BUILD_BITS=64
+BUILD_STYLE=justmake
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME= libxxhash
+COMPONENT_VERSION= 0.8.2
+COMPONENT_SUMMARY= Extremely fast non-cryptographic hash algorithm
+COMPONENT_PROJECT_URL= http://www.xxhash.com/
+COMPONENT_FMRI= library/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION= System/Libraries
+COMPONENT_SRC=		xxHash-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).zip
+COMPONENT_ARCHIVE_URL= https://github.com/Cyan4973/xxHash/archive/refs/tags/v$(COMPONENT_VERSION).zip
+COMPONENT_ARCHIVE_HASH= sha256:a33d7e8798bebb297095b715f93a71c5e535919434ce27ddc65e72e4e0fda3b9
+COMPONENT_LICENSE= BSD 2-Clause License
+
+TEST_TARGET=$(NO_TESTS) # if no testsuite enabled
+include $(WS_MAKE_RULES)/common.mk
+
+COMPONENT_INSTALL_ENV += PREFIX=/usr
+COMPONENT_INSTALL_ENV += LIBDIR=/usr/lib/amd64
```

Comment:
We usually use constants from shared-macros.mk here; $(USRDIR)

Reply:
I got them working.